src/modules/claims/claims.service.ts

```diff
@@ -423,6 +425,15 @@ export class ClaimsService {
           ...(issuerFields && { issuerFields }),
         },
       };
+      if (!credentialStatus) {
+        const credential = this._verifiableCredentialService.createCredential({
+          id: sub,
+          namespace: role,
+          version: version.toString(),
+          issuerFields,
+        });
+        credentialStatus = (await this._addStatusToCredential(credential)).credentialStatus;
```

Comment:
```suggestion
        const credentialStatus = (await this._cacheClient.addStatusToCredential(credential)).credentialStatus;
```
Do not override incoming parameters especially when it's not a primitive type

Reply:
Do we need `addStatusToCredential` method in claim service, we can just use that one from cache client